### ISSUE TITLE
Option (or redesign) to start playing station on short tap instead of long press

### ISSUE THREAD

**Issue by c00ller** [y20k/transistor#71]
I don't see the utility in a short tap revealing a screen with only one possible subsequent option: pressing the Play button. Why would anyone tap on a station other than to start playing it? I think what would make the most sense is to just play the station on short tap. 

This would address another issue I believe the app has, which is the current inconvenience of changing stations. It requires too many taps: pressing the back button to return to the list of stations, tapping on another station, then finally pressing the play button. Going through these many button presses and screen changes is pretty difficult to do while driving.

I realize this implementation would raise other questions, such as how to otherwise get to a "now playing" UI with stop/resume controls, where to show the metadata etc. My only thought is to otherwise make it like Google Play Music where this information and these controls show in a bottom control bar. But I believe the current behavior is just a bit flawed.

**Comment by y20k** [comment]
Hey c00ller,

thanks for your suggestion. I was thinking along similar lines, when I first build the UI for Transistor. Transistor could definitely be a single-view app. At least in its current state. The detail-view can seem redundant. And since the goal of Transistor is to be simple that could have been a way to go. (There might be a visual problem with populating the list with too many play/stop buttons)

On the other hand: Listview->Detailview is a very common pattern. Users almost do expect a Detail View when there is a list. Also having those views makes it easy to create a simple tablet UI. Keeping the Detail View, at least for now, leaves the door open for more station specific features in the future.

> My only thought is to otherwise make it like Google Play Music where this information and these controls show in a bottom control bar. 

Two thoughts about those so-called "mini players" on the bottom of a view. 1st: I do not like them, although they are popular. I do not like the idea of having two places in the UI where you can press a play button. 2nd: Coming from the complexity angle it would be a bit to much to have a list, a mini player and a now playing view.

So to sum it up: I will keep the Detail View ("Now playing") around for the foreseeable future. But I will review this decision when I think that Transistor is feature complete. If the UI then sill seems clumsy or flawed I might consider stripping it down even further.

**Comment by y20k** [comment]
This issue was partly addressed (and solved) here [Make "Now Playing" screen more modern and info-focused ](https://github.com/y20k/transistor/issues/136)
